### ISSUE TITLE
File to import not found or unreadable: bourbon.

### ISSUE THREAD

**Issue by AndreiMotinga** [thoughtbot/bourbon#972]
### Actual behavior
```
ActionView::Template::Error (File to import not found or unreadable: bourbon.
Load paths:
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/babel-source-5.8.35/lib
  /Users/andrei/dev/ezworks/ppn/app/assets/config
  /Users/andrei/dev/ezworks/ppn/app/assets/images
  /Users/andrei/dev/ezworks/ppn/app/assets/javascripts
  /Users/andrei/dev/ezworks/ppn/app/assets/stylesheets
  /Users/andrei/dev/ezworks/ppn/vendor/assets/javascripts
  /Users/andrei/dev/ezworks/ppn/vendor/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/neat-1.8.0/app/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/rails_admin-1.1.0/app/assets/images
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/rails_admin-1.1.0/app/assets/javascripts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/rails_admin-1.1.0/app/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/rails_admin-1.1.0/vendor/assets/javascripts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/rails_admin-1.1.0/vendor/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/remotipart-1.3.1/vendor/assets/javascripts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/nested_form-0.3.2/vendor/assets/javascripts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/jquery-ui-rails-5.0.5/app/assets/images
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/jquery-ui-rails-5.0.5/app/assets/javascripts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/jquery-ui-rails-5.0.5/app/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/font-awesome-rails-4.7.0.0/app/assets/fonts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/font-awesome-rails-4.7.0.0/app/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/normalize-rails-3.0.3/vendor/assets/stylesheets
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/jquery-rails-4.2.1/vendor/assets/javascripts
  /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actioncable-5.0.0.1/lib/assets/compiled
  /Users/andrei/dev/ezworks/vendor/assets/stylesheets):
     6:     %meta{content: "width=device-width, initial-scale=1", name: "viewport; charset=utf-8"}
     7:     %meta{content: "NONE,NOARCHIVE", name: "robots"}
     8:     = csrf_meta_tag
     9:     = stylesheet_link_tag "rails_admin/rails_admin.css", media: :all
    10:     = javascript_include_tag "rails_admin/rails_admin.js"
    11:     -# Initialize JS simple i18n
    12:     :javascript

app/assets/stylesheets/application.scss:5
  Rendering /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/template_error.html.erb within rescues/layout
  Rendering /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_source.html.erb
  Rendered /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_source.html.erb (9.0ms)
  Rendering /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb
  Rendered /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_trace.html.erb (3.5ms)
  Rendering /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_request_and_response.html.erb
  Rendered /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/_request_and_response.html.erb (0.9ms)
  Rendered /Users/andrei/.rvm/gems/ruby-2.3.1/gems/actionpack-5.0.0.1/lib/action_dispatch/middleware/templates/rescues/template_error.html.erb within rescues/layout (65.1ms)
```
### Steps to reproduce behavior
add "rails_admin" gem to app generated with suspenders. 

### Bourbon version
gem 'bourbon', '5.0.0.beta.6'

Gemfile 
```
source "https://rubygems.org"

ruby "2.3.1"

gem "autoprefixer-rails"
gem "country_select"
gem "delayed_job_active_record"
gem "devise"
gem "flutie"
gem "jquery-rails"
gem "normalize-rails", "~> 3.0.0"
gem "pg"
gem "puma"
gem "rack-canonical-host"
gem "rails", "~> 5.0.0"
gem "recipient_interceptor"
gem "sass-rails", "~> 5.0"
gem 'rails_admin', '~> 1.0'
gem "simple_form"
gem "slim-rails"
gem "sprockets", ">= 3.0.0"
gem "sprockets-es6"
gem "suspenders"
gem "title"
gem "uglifier"

group :development do
  gem "listen"
  gem "spring"
  gem "spring-commands-rspec"
  gem "web-console"
end

group :development, :test do
  gem "awesome_print"
  gem "bullet"
  gem "bundler-audit", ">= 0.5.0", require: false
  gem "dotenv-rails"
  gem "factory_girl_rails"
  gem "ffaker"
  gem "pry-byebug"
  gem "pry-rails"
  gem "rspec-rails", "~> 3.5.0.beta4"
end

group :development, :staging do
  gem "rack-mini-profiler", require: false
end

group :test do
  gem "capybara-webkit"
  gem "database_cleaner"
  gem "formulaic"
  gem "launchy"
  gem "shoulda-matchers"
  gem "simplecov", require: false
  gem "timecop"
  gem "webmock"
end

group :staging, :production do
  gem "rack-timeout"
  gem "rails_stdout_logging"
end

gem 'high_voltage'
gem 'bourbon', '5.0.0.beta.6'
gem 'neat', '~> 1.8.0'
gem 'refills', group: [:development, :test]
```

**Comment by tysongach** [comment]
I’m able to reproduce this error, but haven’t figured out why it’s happening.

**Comment by gfauredumont** [comment]
Hi,

I'm not including rails_admin in my Gemfile but Bourbon is having a hard time on my project too.
Don't know if you prefer a new issue or not so for now...:
The Project is OS so you can check [here](https://github.com/gfauredumont/LeProjet/commit/04be5bb2302c3e8e7c2c66f61c3e49ade1ec1315) the commit after which I just added the 'administrate' line in the Gemfile and installed it.

The only point to note in this project is that I removed most of the front libs from the Gemfile and included them with an external Bower (running `bower install` when needed).
I also tried to install Bourbon (v4.2.7) with Bower but no luck either. :/

My error message is the same as everybody else's but I can add detailed logs and error messages if you want.

**Comment by tysongach** [comment]
@gfauredumont Please file a new issue, as this one relates to Bourbon alongside the rails_admin gem.

**Comment by gfauredumont** [comment]
Never mind, the issue I have is with Administrate which hasn't registered the dependency... 
Sorry for the confusion :/

**Comment by vcavallo** [comment]
@tysongach I'm also having the 'Bourbon 5 alongside rails_admin gem' issue - where would be the best place to open that issue? Here or at rails_admin? Thanks.

(for what it's worth, I'm also having issues with Bourbon 5 and administrate, but I've already [commented about that](https://github.com/thoughtbot/administrate/pull/104#issuecomment-272225288) over there.)

**Comment by kbighorse** [comment]
I am also experiencing this issue with ```rails_admin``` and Bourbon 5.0.0.beta.7.

**Comment by tysongach** [comment]
We just published [v5.0.0.beta.8](https://github.com/thoughtbot/bourbon/releases/tag/v5.0.0.beta.8) which potentially solves this issue. Can you try upgrading and let us know if the problem persists or gets resolved?

**Comment by AndreiMotinga** [comment]
no longer use it
